controller_list:
  - name: "effort_joint_trajectory_controller"
    type: "effort_controllers/JointTrajectoryController"
    joints:
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7

  - name: "position_joint_trajectory_controller"
    type: "position_controllers/JointTrajectoryController"
    joints:
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7

  - name: "franka_state_controller"
    type: "franka_control/FrankaStateController"
    joints:
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7

effort_joint_trajectory_controller:
  type: "effort_controllers/JointTrajectoryController"
  joints:
    - panda_joint1
    - panda_joint2
    - panda_joint3
    - panda_joint4
    - panda_joint5
    - panda_joint6
    - panda_joint7
  gains:
    panda_joint1: {p: 100.0, d: 1.0, i: 0.0, i_clamp: 1.0}
    panda_joint2: {p: 100.0, d: 1.0, i: 0.0, i_clamp: 1.0}
    panda_joint3: {p: 100.0, d: 1.0, i: 0.0, i_clamp: 1.0}
    panda_joint4: {p: 100.0, d: 1.0, i: 0.0, i_clamp: 1.0}
    panda_joint5: {p: 100.0, d: 1.0, i: 0.0, i_clamp: 1.0}
    panda_joint6: {p: 100.0, d: 1.0, i: 0.0, i_clamp: 1.0}
    panda_joint7: {p: 100.0, d: 1.0, i: 0.0, i_clamp: 1.0}
  state_publish_rate: 50
  action_monitor_rate: 10

tas_controllers:
  CartesianImpedanceController:
    type: "tas_controllers/CartesianImpedanceController"
    arm_id: panda
    joint_names: 
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7
    stiffness: [1000, 1000, 1000, 100, 100, 100]
    damping: [10, 10, 10, 1, 1, 1]